docs/getting-started/rulesets.md

```diff
@@ -249,6 +249,51 @@ The example above will run all of the rules defined in the `spectral:oas` rulese
 
 - [Rules relevant to OpenAPI v2 and v3](../reference/openapi-rules.md)
 
+## Selectively silencing some results
+
+From time to time, you want to ignore some specific results without turning off 
```

Comment:
Please can we try to pick a word and stick to it? Using both "silencing" and "ignoring" is ok, but then we talk about exceptions.

Can we either `ignore`, or `except`? I think personally think `ignore` is more clear, and in line with `.eslintignore`, `.gitignore`, etc.

Reply:
@philsturgeon Fixed. Do you like it better?

Reply:
Yeah this is great! I'll give it a lick of paint after so you can focus on writing amazing code instead of mucking around with minor tweaks.